install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -49,7 +49,7 @@ data:
 
     {{- if .Values.ingress.enabled }}
```

Comment:
Update this one too `ingress` -> `k8sIngress`

Reply:
I actually see now that we have both `ingress.enabled` and `global. k8sIngress.enabled`.
Confusing.

Reply:
Agreed.  What do you think should be done for 1.1?